src/flick/flick_auth/controllers/register_controller.py

```diff
@@ -39,15 +41,32 @@ def _create_profile(self, profile_data):
         profile.save()
         return profile
 
+    def _check_token(self, token, type="facebook"):
+        """
+        checks if the token is valid, right now only supports facebook token
+        """
+        if not token:
+            return False
+        # only validate if it is turned on from settings, helps with easier development
+        if settings.VALIDATE_SOCIAL_TOKEN:
+            URL = "https://graph.facebook.com/me"
```

Comment:
you should save this URL to `flick/settings.py` and import it here as `settings.VALIDATE_SOCIAL_TOKEN_URL`